src/govuk/components/character-count/character-count.test.js

```diff
@@ -433,6 +433,21 @@ describe('Character count', () => {
           const visibility = await page.$eval('.govuk-character-count__status', el => window.getComputedStyle(el).visibility)
           expect(visibility).toEqual('visible')
         })
+
+        it('injects the description of the textarea', async () => {
```

Comment:
I think this test description needs more context – outside of the context of this PR if this test started failing I'm not sure I'd understand what it was trying to test.